src/ducks/programs.js

```diff
@@ -142,6 +142,55 @@ Effect('getProgram', (data) => {
     }).catch(error => handleError(error));
 });
 
+// For a UI managed by Admins eventually
```

Comment:
Since there's no UI for these Actions _(Effects? Darn this terminology)_ or test code, have we checked if they function properly? I'm OK with merging these if they're not tested, but I'd feel better if we mark them with a "Theoretically should work, check later when UI's built".

Reply:
Yeah, they are untested and theoretically should work, but we don't need to test that until it is needed. I'll add a comment in a future PR.